evtx/Maps/Application_10002.map

```diff
@@ -0,0 +1,65 @@
+Author: Hyun Yi @hyuunnn
+Description: Terminated due to non-response
+EventId: 10002
+Channel: "Application"
+Maps:
```

Comment:
Can you add a Provider for the Application log maps? There are many Providers with overlapping event IDs sometimes so it's best practice to ensure Provider is populated for Application.evtx maps.

Reply:
Fixed :)